src/test/resources/arguments/expected-3.txt

```diff
@@ -0,0 +1,28 @@
+-- Info: src/test/resources/arguments/input.scala:9:4 --------------------------
+9 |  f(???, if(1 == 2) ??? else ???)
+  |    ^^^
+  |    Found hole with type: Int
+-- Info: src/test/resources/arguments/input.scala:9:20 -------------------------
+9 |  f(???, if(1 == 2) ??? else ???)
+  |                    ^^^
+  |                    Found hole with type: String
+-- Info: src/test/resources/arguments/input.scala:9:29 -------------------------
+9 |  f(???, if(1 == 2) ??? else ???)
+  |                             ^^^
+  |                             Found hole with type: String
+-- Info: src/test/resources/arguments/input.scala:12:8 -------------------------
+12 |  ff(1)(???){ ??? }
+   |        ^^^
+   |        Found hole with type: String
+-- Info: src/test/resources/arguments/input.scala:12:14 ------------------------
+12 |  ff(1)(???){ ??? }
+   |              ^^^
+   |              Found hole with type: a.O
+-- Info: src/test/resources/arguments/input.scala:16:4 -------------------------
+16 |  m(???)
+   |    ^^^
+   |
+   |    Found hole with type: Int
+   |    Relevant bindings include
+   |      contextual$1: a.A.OpaqueInt (bound at input.scala:15:4)
```

Comment:
I see this test fails because the compiler-generated name is different in Scala 3.3.x.

Maybe we should have a separate `expected-3.3.txt` file? Or just cheat and remove this case, as it's pretty arcane and it's the only test that fails.

Reply:
I added `expected-3.3.txt`. 😄